### PR TITLE
[admin] Use bootstrap-table in layertree

### PR DESCRIPTION
**Pull request by arnaud-morvan**
Use bootstrap-table from source because of issue fixed but not yet released: https://github.com/wenzhixin/bootstrap-table/issues/3294
Note that jquery-treegrid plugin is no more yet released.

TODO: update tests

### CODE REVIEW

**Comment by arnaud-morvan** [comment]
Here is the new layer tree : localhost_6543_layertree galaxy s5

File: admin/package.json [hunk 1]
```diff
@@ -8,9 +8,9 @@
   "author": "",
   "dependencies": {
     "bootstrap": "3.3.7",
-    "bootstrap-table": "1.11.2",
+    "opencollective": "*",
```
Comment (sbrunner) [inline admin/package.json]: I don't understand what's this fore..., pleas specify the version :-)

Comment (arnaud-morvan) [reply admin/package.json]: Good catch, I will pin the version. We need this to install bootstrap-table from source, it displays a message about donation on stdout during npm install.

Comment (sbrunner) [reply admin/package.json]: OK, thanks :-)

File: admin/c2cgeoportal_admin/static/theme.css [hunk 2]
```diff
@@ -3,14 +3,14 @@
   padding-right: 8px;
 }
 
-#layertree-actions {
-  margin-top: 5px;
-  margin-bottom: 5px;
+
+.bootstrap-table {
+  /* color: #42325b; */
```
Comment (cmangeat) [inline admin/c2cgeoportal_admin/static/theme.css]: dead code, wip

File: admin/c2cgeoportal_admin/templates/layertree.jinja2 [hunk 3]
```diff
@@ -1,117 +1,176 @@
...
 
     <script type="text/javascript">
       $(function() {
-        $('.tree').treegrid({
-          'initialState': 'collapsed',
-          'saveState': true,
-          'saveStateName': 'layertree'
+        var $grid = $('#layertree-table');
+        $grid.bootstrapTable({
+          height: $(window).height() - $('nav').height() - 30,
```
Comment (cmangeat) [inline admin/c2cgeoportal_admin/templates/layertree.jinja2]: 30 should be variabilized ?

**Comment by arnaud-morvan** [comment]
The need was to have same visual aspect as other grids (not for functional needs but visual consistency).
Note that this bring base factorization for action sharing between layertree and grids/forms.

**Pull request merged**